src/main/js/helpers/axis.js

```diff
@@ -577,9 +577,6 @@ const getYAxisHeight = (config) => config.height;
  * @returns {number} Height of the X Axis ticks, labels and numbers/datetimes
  */
 const getXAxisHeight = (config) => {
-    if (config.padding.hasCustomPadding) {
-        return config.padding.bottom;
```

Comment:
If I remember right - removing this might break Gantt with customPadding set to all 0's. Can you post a screenshot of what happens for Gantt having 0 custom padding

Reply:
<img width="1457" alt="Screenshot 2019-09-20 at 11 22 45 AM" src="https://user-images.githubusercontent.com/29273628/65303086-9e5f4100-db9a-11e9-86fc-e668ea171a6e.png">

It doesn't break the code. But I am not able to understand that why are using `tracklabel` to define padding `left` both for `custom` and `constant` [GanttConfig.js](https://github.com/cerner/carbon-graphs/blob/master/src/main/js/controls/Gantt/GanttConfig.js#L133)

Reply:
The reason might be because, track labels are displayed in the left and the length of the trackLabel defines where to start the Gantt (since we wouldn't want our labels to get cut off - giving arbitrary values).